commit b80982fe6035739c1aae7e86a5b1e6de0414a9a6
Author: Apollon Oikonomopoulos <apoikos@debian.org>
Date:   Tue Apr 21 18:29:47 2015 +0300

    Makefile.am: use C.UTF-8
    
    There is no need to use en_US.UTF-8 with newer libc versions. Use
    C.UTF-8 to avoid an extra B-D on locales-all.

diff --git a/Makefile.am b/Makefile.am
index 364b26b..3f64939 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2179,7 +2179,7 @@ man/%.7.in man/%.8.in man/%.1.in: man/%.gen man/footer.rst
 	trap 'echo auto-removing $@; rm $@' EXIT; \
 	$(PANDOC) -s -f rst -t man $< man/footer.rst | \
 	  sed -e 's/\\@/@/g' > $@; \
-	if test -n "$(MAN_HAS_WARNINGS)"; then LC_ALL=en_US.UTF-8 $(CHECK_MAN_WARNINGS) $@; fi; \
+	if test -n "$(MAN_HAS_WARNINGS)"; then LC_ALL=C.UTF-8 $(CHECK_MAN_WARNINGS) $@; fi; \
 	$(CHECK_MAN_DASHES) $@; \
 	trap - EXIT
 
@@ -2816,7 +2816,7 @@ $(APIDOC_HS_DIR)/index.html: $(HS_LIBTESTBUILT_SRCS) Makefile
 	  $(HSCOLOUR) -print-css > $$i/hscolour.css; \
 	done
 	set -e ; \
-	export LC_ALL=en_US.UTF-8; \
+	export LC_ALL=C.UTF-8; \
 	OPTGHC="--optghc=-isrc --optghc=-itest/hs"; \
 	OPTGHC="$$OPTGHC --optghc=-optP-include --optghc=-optP$(HASKELL_PACKAGE_VERSIONS_FILE)"; \
 	for file in $(HS_LIBTESTBUILT_SRCS); do \
